lib/token.h

```diff
@@ -1426,6 +1433,7 @@ class CPPCHECKLIB Token {
         fIsFinalType            = (1ULL << 42), // Is this a type with final specifier
         fIsInitComma            = (1ULL << 43), // Is this comma located inside some {..}. i.e: {1,2,3,4}
         fIsInitBracket          = (1ULL << 44), // Is this bracket used as a part of variable initialization i.e: int a{5}, b(2);
+        fIsAnonymousBitfield    = (1ULL << 45), // Is this a token added for an unnamed bit-field
```

Comment:
how about `fIsAnonymous` and reuse this for other anonymous symbols also.. i.e. unnamed structs.